plugin/core/views.py

```diff
@@ -108,8 +108,17 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
         return linecache.getline(file_name, row + 1).strip()
 
 
+def get_storage_path() -> str:
+    """
+    The "Package Storage" is a way to store server data without influencing the behavior of Sublime Text's "catalog".
+    Its path is '$DATA/Package Storage'.
+    """
+    return os.path.abspath(os.path.join(sublime.cache_path(), "..", "Package Storage"))
```

Comment:
As it was just mentioned on Discord, this won't be correct on Windows since cache is not part of "$DATA" directory there.

Reply:
maybe  `sublime.installed_packages_path()` is? 
I don't have a windows os.

Reply:
```
>>> sublime.installed_packages_path()
'C:\\Users\\<username>\\AppData\\Roaming\\Sublime Text 3\\Installed Packages'
>>> sublime.packages_path()
'C:\\Users\\<username>\\AppData\\Roaming\\Sublime Text 3\\Packages'
>>> sublime.cache_path()
'C:\\Users\\<username>\\AppData\\Local\\Sublime Text 3\\Cache'
```
for a standard installation (not portable) on Windows and ST build 3211.

Reply:
> As it was just mentioned on Discord, this won't be correct on Windows since cache is not part of "$DATA" directory there.

This *is* the correct folder for windows. The roaming appdata folder is for config files, the local appdata is for machine-dependent files.

Reply:
True. I guess that when making a comment I was thinking that the ST cache dir is a direct child of `Local`.